util/pom.xml

```diff
@@ -75,7 +80,7 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-iidm-xml-converter</artifactId>
-            <scope>test</scope>
+            <scope>runtime</scope>
```

Comment:
i don't see where this can be needed in runtime (?)

Reply:
ok